### DISCUSSION TITLE
Breakout Commits into separate Pull Requests

### DISCUSSION THREAD

**Discussion by ustireman** [community/community#111986] [New to GitHub]
A really new user of Github. I have been just used mainly  the Website but have the desktop app installed as well. I have used the CLI in a very limited capacity through VS Code . I recently contributed to a Repo that I first forked, and then created a branch . I uploaded and committed some files to the Repo and created a pull request. I later on uploaded and committed additional files but to the same branch and linked to the same pull request. The project maintainer has recently requested that I break up the commits into different Pull requests for him to be able to review and merge. I have searched but can't seem to find the right way to do this basically this is what I did
B  -->  is my Forked Repo 
   C--> Branch in B
   PR1 ----> Initial Pull Request after commits to C;
                  CM1
                  CM2
Later I  added additional commits
                 CM3
                 CM4
                 CM5
                 CM6
CM3, CM4,CM5 and CM6 are all now linked to PR1
 What I think the project maintainer wants is to have something like this
PR1 -- > CM1
PR2 --> CM2
PR3 --> CM3
PR4 --> CM4
PR5 --> CM5
PR6 --> CM6
How do I go about either reverting my commits and then creating separate Pull Requests for each?

Thanks

**Comment by mymarius** [comment]
Yes, but how?

**Reply by ustireman** [reply]
I was hoping that I could get an answer on this forum that is why I posted the question. Is this the wrong place to post this ? Can you direct me to the right place?
